src/nupic/algorithms/KNNClassifier.py

```diff
@@ -259,6 +259,7 @@ def _sparsifyVector(self, inputPattern, doWinners=False):
     if doWinners:
       if (self.numWinners>0) and (self.numWinners < (inputPattern > 0).sum()):
         sparseInput = numpy.zeros(inputPattern.shape)
+        
```

Comment:
I don't know what you mean could you be more clear? I don't see anything but perfectly formatted lines?